spec/Section 6 -- Execution.md

```diff
@@ -154,7 +155,8 @@ ExecuteMutation(mutation, schema, variableValues, initialValue):
 - Let {selectionSet} be the top level Selection Set in {mutation}.
 - Let {data} be the result of running {ExecuteSelectionSet(selectionSet,
   mutationType, initialValue, variableValues)} _serially_.
-- Let {errors} be any _field errors_ produced while executing the selection set.
+- Let {errors} be the list of all _field error_ raised while executing the
```

Comment:
field errors plural?

Reply:
That won’t work with the definitions feature of spec-md